core/controllers/resources_test.py

```diff
@@ -60,9 +65,7 @@ def test_image_upload_and_download(self):
 
         self.logout()
 
-        response = self.testapp.get(
-            str('%s/0/assets/image/%s' %
-                (self.IMAGE_VIEW_URL_PREFIX, filepath)))
+        response = self.testapp.get(self._get_image_url('0', filepath))
```

Comment:
Done.

Reply:
Hm. If you look at the usage of this part of the URL throughout the frontend tests, it's referred to as a filename. So there is still a discrepancy.

Is it a filename or a filepath? It's looking more like a filename to me, but please verify and confirm.

Reply:
Hi Sean sorry for the late response.

Yes you're right it seems all the frontend tests are referring to this as filename. I think the resource handler is able to take a file path as param, which is probably why it was referred to as encoded_filepath. Do you think it makes sense to start using filename everywhere, since I can't find any use of file path in our code so far?

Reply:
That sounds good!

So I've changed these to filename as well as the response for ImageUploadHandler to also use filename instead of filepath. The AudioUploadHandler class is already using filename.